index.md

```diff
@@ -3,37 +3,45 @@ title: Sysdig Helm Charts
 description: Repository for Sysdig's Helm charts.
 ---
 
-## Add this charts repository to your local Helm
+Helm is an open-source package manager for Kubernetes applications. Using a helm chart you can describe, install, configure, and upgrade applications automatically.
 
-To start using our charts, first add this _charts repository_ using the `helm` command line tool:
+To simplify your deployment experience, Sysdig provides helm charts to install and configure Sysdig installation components in your Kubernetes environment.
+
+
+## Sysdig Chart Repository
+
+The Sysdig Chart Repository includes the following charts:
```

Comment:
due to removing grammatical list of charts, i would maintain deprecated charts too but in the lower part of the page, and marked as deprecated.